app/controllers/events/view/settings.js

```diff
@@ -3,6 +3,34 @@ import { action } from '@ember/object';
 
 export default class extends Controller {
 
+  @action
+  openDeleteEventModal(id, name) {
+    this.setProperties({
+      'isEventDeleteModalOpen' : true,
```

Comment:
Why setting for properties? Revert the duplication

Reply:
@kushthedude done :+1: